README.md

```diff
@@ -1 +1,47 @@
 # provider-in-cluster
+
```

Comment:
@krishchow I have some general comments for this README - remember our goal is to make this easily consumable for any level of user

- I think we need a general summary or description of the provider at the top before you jump into the prereqs - what does it do?
- You currently support two examples of using this operator - OLM/Operators and Database/Postgres - it might be nice to organize or restructure the flow a bit 
     - Getting Started section (within that, explain there are two working examples and what they do, brief description)
     - You can off shoot from that section to separate readme's for each example or just explain they are below with 
        with a link to them

-  I think each section needs more meat and more exact steps on how to run this - similar to what you did for ET repo
-  Database section doesn't point to anything, should be same as OLM example

Let me know if I'm making sense, we can discuss off line as well

Reply:
Sounds good Scott, I updated the documentation as described in your comment :)

Reply:
@krishchow - getting there, a few more nits

1. I think it's cleaner to have commands inside of code blocks ``` ... ```  and on separate lines, not embedded in text
2. Under the examples sections you have, I'd like to see a step by step... 1. do this, 2. do this, 3. do this - all with the commands, including the oc/kubectl commands that are run. You reference the examples directory but you should help the user decipher which files they need to use to run the example, what commands for each file, etc... step by step guide for dummies

Reply:
@screeley44 - should be ready for another review, I have made the changes as requested :smile: